khmer/_khmermodule.cc

```diff
@@ -1541,10 +1618,14 @@ static PyMethodDef khmer_counting_methods[] = {
     { "hashsizes", (PyCFunction)hash_get_hashsizes, METH_VARARGS, "" },
     { "set_use_bigcount", (PyCFunction)hash_set_use_bigcount, METH_VARARGS, "" },
     { "get_use_bigcount", (PyCFunction)hash_get_use_bigcount, METH_VARARGS, "" },
+    { "init_threadsafe", (PyCFunction)hash_init_threadsafe, METH_VARARGS, "Initialize the table lock array" },
+    { "n_lock_blocks", (PyCFunction)hash_n_lock_blocks, METH_VARARGS, "Number of blocks in the lock array" },
```

Comment:
The number of lock blocks is passed when calling `init_threadsafe`. It has a default value that is hardcoded, but the user can override it; there isn't necessary anything private or testing-specific about it. I will make it const const though.